apps/prairielearn/src/sprocs/sync_course_instances.sql

```diff
@@ -154,7 +154,6 @@ BEGIN
         json_comment = (src.data->>'comment')::jsonb,
         self_enrollment_enabled = (src.data->>'self_enrollment_enabled')::boolean,
         self_enrollment_enabled_before_date = input_date(src.data->>'self_enrollment_enabled_before_date', COALESCE(src.data->>'display_timezone', c.display_timezone)),
-        self_enrollment_enabled_before_date_enabled = (src.data->>'self_enrollment_enabled_before_date_enabled')::boolean,
```

Comment:
AFAICT this value isn't meaningfully read anywhere (especially anywhere that's not currently behind a feature flag), so I'm dropping this in this PR.